tox.ini

```diff
@@ -18,7 +19,7 @@ deps =
 
 [testenv:lint]
 commands =
-    -flake8
+    flake8 --max-line-length=120 --ignore=C901,N801,N802,N803,N806,N816
```

Comment:
Configuration for black sets this to 100, let's use the same here

Reply:
https://github.com/ubuntu/microk8s/blob/master/pyproject.toml#L2

Reply:
TIL that black does not treat the line length limits as hard limits. In [1] it says:

> If you’re paid by the line of code you write, you can pass --line-length with a lower number. Black will try to respect that. However, sometimes it won’t be able to without breaking other rules. In those rare cases, auto-formatted code will exceed your allotted limit.

So what we are doing here is we say to black to target 100 chars but we still have exceptions to that limit that flake8 can report:
```
$ sudo tox -e lint      
lint installed: appdirs==1.4.4,attrs==21.4.0,black==21.4b2,cachetools==5.0.0,certifi==2021.10.8,cffi==1.15.0,ch.....
lint run-test-pre: PYTHONHASHSEED='1713914300'
lint run-test: commands[0] | flake8 --max-line-length=100 --ignore=C901,N801,N802,N803,N806,N816
./scripts/cluster/join.py:55:101: E501 line too long (116 > 100 characters)
./scripts/cluster/join.py:784:101: E501 line too long (102 > 100 characters)
./scripts/cluster/join.py:787:101: E501 line too long (101 > 100 characters)
./scripts/cluster/join.py:797:101: E501 line too long (120 > 100 characters)
./scripts/cluster/join.py:815:101: E501 line too long (105 > 100 characters)
./scripts/cluster/add_token.py:96:101: E501 line too long (117 > 100 characters)
./scripts/wrappers/reset.py:38:101: E501 line too long (101 > 100 characters)
./scripts/wrappers/reset.py:124:101: E501 line too long (106 > 100 characters)
./scripts/wrappers/common/utils.py:178:101: E501 line too long (101 > 100 characters)
ERROR: InvocationError for command /home/jackal/workspace/microk8s/.tox_env/bin/flake8 --max-line-length=100 --ignore=C901,N801,N802,N803,N806,N816 (exited with code 1)
```

So the hard limit is enforced by flake8 and is set to 120 chars.

 
[1] https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length